crates/sui-network/src/state_sync/mod.rs

```diff
@@ -893,6 +965,11 @@ where
                     {
                         // peer didn't give us a checkpoint with the height that we requested
                         if *checkpoint.sequence_number() != next {
+                            tracing::error!(
```

Comment:
this should not be an error! since it does not necessarily indicate a bug on our side, any byzantine peer could do this on purpose. it should be info! at most (probably debug?)